web/src/app/features/oauth2/components/oauth2-base-page/oauth2-base.component.html

```diff
@@ -0,0 +1,64 @@
+<div
+  class="min-h-screen flex flex-col items-center justify-center bg-secondary p-4">
+  <div class="w-full max-w-md bg-white shadow-md rounded-lg p-8 text-center">
+    <div class="flex flex-col items-center">
+      <img
+        ngSrc="icons/triggo_logo.svg"
+        alt="Triggo Logo"
+        [width]="100"
+        [height]="100"
+        class="mb-3" />
+      <h1 class="text-2xl font-bold tr-title-2 mb-6">
+        {{ name() }} Integration
+      </h1>
+    </div>
+
+    @if (!loading && !success()) {
+      <div class="bg-red-50 border border-red-200 rounded-lg p-4 text-center">
+        <ng-icon
+          name="heroExclamationTriangle"
+          size="2.5rem"
+          class="text-red-400" />
+
+        <h2 class="tr-large font-semibold text-red-800 mb-2">
+          Integration Failed
+        </h2>
+        <p class="text-red-600 text-sm">
+          We couldn't complete the {{ name() }} integration. Please try again or
+          contact support.
+        </p>
+      </div>
+    }
+    @if (!loading() && success()) {
+      <div class="space-y-4">
+        <div
+          class="bg-green-50 border border-green-200 rounded-lg p-4 text-center">
+          <ng-icon
+            name="heroCheckCircle"
+            size="2.5rem"
+            class="text-green-400" />
+
+          <h2 class="tr-large text-green-800 mb-2">
+            {{ name() }} Connected Successfully
+          </h2>
+          <p class="text-green-600 mb-4 text-sm">
+            Your {{ name() }} account has been integrated.
```

Comment:
I would say linked but anyway.
```suggestion
            Your {{ name() }} account has been linked.
```